site/ts/grader-submission.ts

```diff
@@ -20,7 +20,37 @@ declare global {
 
 const warning_banner = document.getElementById('submission-mode-warning');
 
+
+function initialSubmissionMode() {
+    const radioBulk = $('#radio-bulk');
+    const pdfSubmitButton = $('#pdf-submit-button');
+    const warningBanner = $('#warning-banner');
+
+    if (radioBulk.length) {
+        if (radioBulk.prop('checked')) {
+            if (pdfSubmitButton.length) {
+                pdfSubmitButton.show();
+            }
+            sessionStorage.setItem(`${window.gradeable_id}-submission_mode`, 'bulk-upload');
+
+            if (warningBanner.length && warningBanner.children().length) {
+                const message = 'Warning: Submitting files for bulk upload!';
+                warningBanner.children().first().text(message);
+            }
+        }
+        else {
+            if (pdfSubmitButton.length) {
+                pdfSubmitButton.hide();
+            }
+        }
```

Comment:
This can be combined into a single elseif:
```suggestion
        else if (pdfSubmitButton.length) {
            pdfSubmitButton.hide();
        }
```